chef-config/lib/chef-config/config.rb

```diff
@@ -65,7 +65,7 @@ def self.platform_specific_path(path)
         # Some installations will be on different drives so use the drive that
         # the expanded path to __FILE__ is found.
         drive = windows_installation_drive
-        if drive && path[0] == '\\' && path.split('\\')[2] == "chef"
+        if drive && path[0] == '\\' && path.split('\\')[2] == Chef::Dist::Exec
```

Comment:
while at the moment this path is the exec name we shouldn't count on that. We could change the executable name and this dir would stay the same ideall.

Reply:
I'll make a Chef::Dist::WIN_CONF_DIR

Reply:
I'll need to wait for https://github.com/chef/chef/pull/9060 to be merged, the new constant should go in `ChefConfig::Dist` but #9060 creates it.